articles/tokens/guides/configure-refresh-token-rotation.md

```diff
@@ -0,0 +1,71 @@
+---
+description: Learn how to configure Refresh Token rotation.
+topics:
+  - tokens
+  - refresh-tokens
+  - refresh-token-rotation
+contentType: how-to
+useCase:
+  - enable-refresh-token-rotation
+  - configure-refresh-token-rotation
+---
+# Configure Refresh Token Rotation
+
+Configure Refresh Token Rotation for each application using the Management API. When Refresh Token Rotation is enabled, the transition for the end-user is seamless. The application uses the previous non-rotating Refresh Token, which has expired and swaps it for a rotating Refresh Token. 
+
+::: note
+Migration scenarios accommodate automatic token revocation when migrating from a non-rotating Refresh Token to a rotating Refresh Token and vice-versa.
+:::
+
+Request a new Refresh Token/Access Token pair when Refresh Token Rotation is enabled.
+
+- Exchanging a non-rotating Refresh Token when Refresh Token Rotation is enabled deletes all the non-rotating tokens issued for the same `client_id`, resource server, and user and tenant.
+- Exchanging a rotating Refresh Token when Refresh Token Rotation is disabled issues a non-rotating Refresh Token and revokes the Rotating Refresh Token family.
+
+The following steps describe how to configure Refresh Token rotation for a SPA, as an example. 
+
+1. Install the latest version of the `auth0-spa-js` SDK.
+
+    ```text
+    npm install @auth0/auth0-spa-js
+    ```
+
+2. Use the Managment API to enable Refresh Token rotation:
```

Comment:
Will this be required? Presumably by the time these docs hit live, the changes to Manhatten will be live and the admin will be able to enable this feature using the UI rather than using the management API.

Reply:
Exactly. I also shared some screenshots of what the Admin controls look like and suggest that this be updated.